sign_csr.py

```diff
@@ -2,7 +2,43 @@
 import argparse, subprocess, json, os, urllib2, sys, base64, binascii, time, \
     hashlib, tempfile, re, copy, textwrap
 
-def sign_csr(pubkey, csr, email=None):
+
+def host_token(domain, token, response_payload, user_step_number, file_based):
+    if file_based:
+        response_url = 'http://{}/.well-known/acme-challenge/{}'.format(domain, token)
+
+        # tell the user where to put a file
+        sys.stderr.write("""\
+STEP {}: You need to place a file on your webserver so that the URL {} will
+resolve to a file with the contents:
+
+--------------
+{}
+--------------
```

Comment:
I've updated the text including notes to clarify how the file must be formatted (no whitespace).